addons/website_project/models/project_task.py

```diff
@@ -1,13 +1,9 @@
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import api, models, fields
```

Comment:
is it not part of odoo anymore :point_right: :point_left:

Reply:
I saw the unused api import, and couldn't resist doing a tiny bit of cleanup :innocent: